cmd/kubeadm/app/cmd/init.go

```diff
@@ -107,7 +107,7 @@ var (
 
 // NewCmdInit returns "kubeadm init" command.
 func NewCmdInit(out io.Writer) *cobra.Command {
-	cfg := &kubeadmapiext.MasterConfiguration{}
+	cfg := &kubeadmapiext.MasterConfiguration{ClusterName: kubeadmapiext.DefaultClusterName}
```

Comment:
Is `ClusterName: kubeadmapiext.DefaultClusterName` needed to be specified here? I think it will be defaulted to the value by `legacyscheme.Scheme.Default`.

Reply:
You're right. Did not know that's what legacyscheme did. Thanks